README.md

```diff
@@ -4,6 +4,10 @@
 
 Developing a common ontology for energy system analysis and scenarios.
 
+## Scope of this ontology
+
+This **domain** ontology is a joint effort to represent the energy-system modelling context based on standard terminologies used by human experts in this field of research. It is designed to improve transparency and facilitate comparability & transferability within the energy system modelling context.
```

Comment:
Maybe we should also add 

This ontology makes use of the Basic Formal Ontology (BFO) principles. 

and then add a clickable link to BFO (as below). Then the scope also includes information on the principles right away.